internal/evaluator/rego.go

```diff
@@ -118,6 +159,30 @@ func ociBlob(bctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
 	return ast.StringTerm(blob.String()), nil
 }
 
+func purlIsValid(bctx rego.BuiltinContext, a *ast.Term) bool {
```

Comment:
This must implement this signature:

```go
Builtin1 func(bctx BuiltinContext, op1 *ast.Term) (*ast.Term, error)
```

It's missing an error parameter (which will always be nil :disappointed:)

Reply:
Done!